lib/computer_player.rb

```diff
@@ -1,20 +1,17 @@
-require_relative 'board_tracker'
-
 class ComputerPlayer < Player
 
-  def initialize(player_num, board_array)
-    @board_array = board_array
-    @board_tracker = BoardTracker.new(@board_array)
+  def initialize(player_num, board)
+    @board = board
   end
 
   def get_next_move
-    available_moves = @board_tracker.get_available_moves
+    available_moves = @board.get_available_moves
```

Comment:
It seems awkward to me that the player has an instance of the board.  It might make more sense to pass in the board as an argument to this method.

Reply:
I agree. The player should only need information about the board when picking its next move. I've made the appropriate changes.